src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 50px;
+
+}
+
+.nav__link {
+  display: flex;
+  height: 60px;
+  position: relative;
+  text-decoration: none;
+  color: black;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: calc(100% - 12px) 0;
+  margin: 20;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  list-style-type: none;
+
+  margin: 0;
+}
+
+.nav__link:hover {
+  color: #00ACDC;
+  display: flex;
+}
+
+.logo {
+  margin: 0;
+  padding: 0;
+  border: none;
+  vertical-align: middle;
+}
+
+.is-active {
+  position: relative;
+  display: block;
+  color:#00ACDC;
+  text-decoration: none;
+  padding-top: 22.7;
+  padding-bottom: 22.7;
```

Comment:
you may add such padding to the `nav_link` - in this case, the clickable area of the link will be bigger, which is better for UX. But make sure the number is correct (should pass tests).

Reply:
I passed the test, I can’t upload a new link here